recipes/cpprestsdk/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "2.10.14":
+    sha256: f2628b248f714d7bbd6a536553bc3782602c68ca1b129017985dd70cc3515278
+    url: https://github.com/Microsoft/cpprestsdk/archive/v2.10.14.tar.gz
```

Comment:
```suggestion
```
You can exclude the old version, since it's a patch

Reply:
picked up from your pr